### PR TITLE
Fix 2.6.x semaphore

### PR DESCRIPTION
**Pull request by tmjd**

### CODE REVIEW

File: calico_node/Makefile [hunk 1]
```diff
@@ -527,8 +527,8 @@ semaphore:
 	# Actually run the tests (refreshing the images as required), we only run a
 	# small subset of the tests for testing SSL support.  These tests are run
 	# using "latest" tagged images.
-	$(MAKE) RELEASE_STREAM=master $(NODE_CONTAINER_NAME) st
-	ST_TO_RUN=tests/st/policy $(MAKE) RELEASE_STREAM=master st-ssl
+	$(MAKE) RELEASE_STREAM=$(RELEASE_STREAM) $(NODE_CONTAINER_NAME) st
```
Comment (caseydavenport) [inline calico_node/Makefile]: Ah, right.

Yeah, this is OK. We could even hard-code the release stream to v2.6 here, given this is going into the v2.6.x-series branch.

**Comment by tmjd** [comment]
These tests aren't going to pass until _data/versions.html is updated to pull in the next release versions of felix, calicoctl, cni, and maybe others.  Due to ApplyOnForward being added yet currently the tests are pulling in the v2.6.2 versions.

**Comment by fasaxc** [comment]
Apply-on-forward isn't in in 2.6.x, but looks like we cut the v2.6.2 tag from master after we'd added tests (for then-master Felix).  I'll see if I can put together a branch that takes this and omits those tests...

**Comment by fasaxc** [comment]
Picked up under #1395

**Pull request closed without merging**